### DISCUSSION TITLE
四月更新 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Clean-Reader/CleanReader.Desktop#19] [Announcements]
## 🐛 Bug 修复

- 修复阅读时长显示异常的问题
- 修复发现页面点击顶部溢出按钮会导致应用崩溃的问题 @Richasy (#17)
- 修复发现页面加载时窗口失去响应的问题 @Richasy (#16)
- 替换字体获取方式 @Richasy (#15)
- 修复平滑滚动下文本渲染错位的问题 @Richasy (#14)

<hr /><em>This discussion was created from the release <a href='https://github.com/Clean-Reader/CleanReader.Desktop/releases/tag/v3.2204.3'>四月更新 - 2</a>.</em>

**Comment by Curious-r** [comment]
好快的修复
